app/views/api/guidance/release_notes.yml

```diff
@@ -20,6 +23,9 @@
 
 - title: View financial statements in test environment
   date: 2025-06-23
+  tags:
+    - "#sandbox-release"
+    - "#new-feature"
```

Comment:
I'd be tempted to drop the `#` prefix here, then the `.tr("#", "")` above can be removed and the prefix from each `TAG_MAPPING` entry.